dev-perl/Statistics-Basic/Statistics-Basic-1.661.1.ebuild

```diff
@@ -14,5 +14,6 @@ KEYWORDS="~amd64 ~ia64 ~ppc ~sparc ~x86"
 IUSE=""
 
 DEPEND="dev-perl/Number-Format"
+RDEPEND="${DEPEND}"
```

Comment:
Also this is in a wrong commit now, it belongs to the previous one. You can reset your branch, then wget files from this PR to start over fresh (it's faster than separating commits, then rebasing manually)